client/src/EstimatesComparison/EstimatesExplorerComponent.js

```diff
@@ -64,12 +64,12 @@ const DetailedAmountsByDoc = ({ amounts_by_doc }) => {
                 <tr key={doc_code}>
                   <td>{estimates_docs[doc_code][lang]}</td>
                   <td>
-                    {amount_last_year && (
+                    {(amount_last_year || amount_last_year === 0) && (
                       <Format type="compact2" content={amount_last_year} />
                     )}
                   </td>
                   <td>
-                    {amount_this_year && (
+                    {(amount_this_year || amount_this_year === 0) && (
```

Comment:
This is kinda sketchy but theoretically this should be set to `false`.....
<img width="574" alt="Screen Shot 2021-02-18 at 12 04 18 PM" src="https://user-images.githubusercontent.com/29746626/108393437-aa14c400-71e1-11eb-89a6-d0425f3c050f.png">

Reply:
Aren't these the cases where there's some non-zero data, but the net value is 0 because they cancel out? I think that's the point of that check, we only want 0s if they're because the net value is zero. If ALL values are zero then we want to display it as blank/what used to be inactive (except inactive was always being accidentally applied to the net-zero case as well).

Reply:
I don't think these checks were for net value of zero.. because I think this is strictly for table of breakdown. The status (active or inactive) is determined by `current_value`